frontend/src/components/layout/ProgressLoader.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+import ProgressBar from 'primevue/progressbar';
+
+import { useAppStore } from '@/store';
+
+const appStore = useAppStore();
+const { getLoadingMode, getLoadingValue } = storeToRefs(appStore);
+</script>
+
+<template>
+  <!-- eslint-disable vue/attribute-hyphenation -->
+  <ProgressBar
+    :mode="getLoadingMode"
+    :showValue="false"
+    :value="getLoadingValue"
+  />
```

Comment:
Is this disable necessary? Was it not correctly accessing the PrimeVue attribute when hyphenated?

Reply:
You are right we can do it in hyphenated style and it still works.